docker-alfresco/test/docker-alfresco-all-amps-test/Dockerfile

```diff
@@ -49,4 +49,4 @@ RUN mkdir /usr/local/tomcat/webapps/ROOT && cd /usr/local/tomcat/webapps/ROOT &&
 # Add catalina.policy to ROOT.war and alfresco.war
 # Grant all security permissions to alfresco webapp because of numerous permissions required in order to work properly.
 # Grant only deployXmlPermission to ROOT webapp.
```

Comment:
add a comment here about adding permissions to jolokia and share for the all-amps-docker image - it may help in the future to better understand that sed command